openwisp-config/files/openwisp.agent

```diff
@@ -370,6 +380,28 @@ get_checksum() {
 		return 2
 	fi
 
+	if is_http_status "$1" 404; then
+		for attempt_no in $(seq 0 "$CHECKSUM_MAX_RETRIES"); do
```

Comment:
setting seq 0 instead of seq 1 seems to resolve the issue below (https://github.com/openwisp/openwisp-config/pull/163/files#r830518052)

Reply:
By setting it to zero it will make checksum request `CHECKSUM_MAX_RETRIES+1` times.

Reply:
you're right, I must have misunderstood logging in my testing, thanks